#this service is for updating the tourist point map

tp_msg[] miniverse

---

tp_msg[] miniverse